client/wp-admin-scripts/payment-method-promotions/payment-promotion-row.tsx

```diff
@@ -103,24 +115,37 @@ export const PaymentPromotionRow: React.FC< PaymentPromotionRowProps > = ( {
 		);
 	};
 
+	const onDismiss = () => {
+		setIsVisible( false );
+		updatePaymentGateway( gatewayId, {
+			settings: {
+				is_dismissed: 'yes',
+			},
+		} );
```

Comment:
I believe [Pedro would also like a track to be triggered here](https://github.com/woocommerce/woocommerce-admin/issues/7951#issuecomment-976372869), I am not sure what we would call it.

Reply:
The recommended card below triggers this track `settings_payments_recommendations_dismiss`, so maybe we should do a `settings_payments_promotions_dismiss`? with the plugin name as a prop (although it will probably only be WC Pay for the foreseeable future).

Reply:
Good call!  I missed that comment.  Added in https://github.com/woocommerce/woocommerce-admin/pull/7965/commits/4d9e28148598beef56797d92e96d8acfb38c88f9